packages/editor-ui/src/components/RunInfo.vue

```diff
@@ -50,27 +50,44 @@ const runMetadata = computed(() => {
 			"
 		></span>
 	</n8n-info-tip>
-	<n8n-info-tip
-		v-else-if="runMetadata"
-		type="tooltip"
-		:theme="theme"
-		:data-test-id="`node-run-info-${theme}`"
-		tooltip-placement="right"
-	>
-		<div>
-			<n8n-text :bold="true" size="small"
-				>{{
-					runTaskData?.error
-						? i18n.baseText('runData.executionStatus.failed')
-						: i18n.baseText('runData.executionStatus.success')
-				}} </n8n-text
-			><br />
-			<n8n-text :bold="true" size="small">{{ i18n.baseText('runData.startTime') + ':' }}</n8n-text>
-			{{ runMetadata.startTime }}<br />
-			<n8n-text :bold="true" size="small">{{
-				i18n.baseText('runData.executionTime') + ':'
-			}}</n8n-text>
-			{{ runMetadata.executionTime }} {{ i18n.baseText('runData.ms') }}
-		</div>
-	</n8n-info-tip>
+	<div v-else-if="runMetadata" :class="$style.tooltipRow">
```

Comment:
Please advise on this `div`. Is this the proper way to do "two elements in a row" in our codebase? Also please advise on existing standards for column-gap/padding between icons in the class added below. I tried `var(--spacing-xs)` but the gap ended up too large.

Reply:
Yes, using `flex` and `gap` (or `column-gap`) is the way to go.
Regarding the spacing: If you take a look at our storybook, `spacing-xs` resolves to `12px`, so the correct value to use here is `spacing-4xs`.

Reply:
Ah silly me, 4xs is smaller than xs. I'm the reason the 1/3 pound burger failed 🫨